src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -269,4 +269,23 @@ public CardGetDTO getCurrentBlackCard(@RequestHeader(value = "authorization", re
     BlackCard blackCard = userService.getCurrentBlackCard(userId);
     return DTOMapper.INSTANCE.convertEntityToCardGetDTO(blackCard);
   }
+
+    /**
+     * Gets the current black card of a user or throws status exception
+     * @param token: token of the user
+     * @param userId: userId of the user
+     * @return: GameGetDTo
+     */
+    @GetMapping("/users/{userId}/games/activeGame")
```

Comment:
Have you thought about extending this URI in a way that the client can decide what games to get (/users/userId/games?active=true). [Thinking loud]

Reply:
can do.
- Should we then change the getGame url? (which returns the most ecent game /users/{userId}/games/vote)
- or do you want to have an url to only get pastGames? For what?

Reply:
Maybe we shouldn't do it, because active = true would get a list with one element and active = false would return very much data.

Reply:
true that. and there is not realy a reason for getting past games